decidim-system/lib/decidim/system/menu.rb

```diff
@@ -10,6 +10,11 @@ def self.register_system_menu!
                         decidim_system.root_path,
                         position: 1,
                         active: ["decidim/system/dashboard" => :show]
+          menu.add_item :api_credentials,
+                        I18n.t("menu.api_credentials", scope: "decidim.system"),
+                        decidim_system.api_users_path,
+                        position: 2,
+                        active: ["decidim/system/api_users"]
```

Comment:
This route seems to be missing if the `Api` module is missing, so this statement should consider the same constraint as we have for adding the route.